roles/label_nodes/README.md

```diff
@@ -0,0 +1,42 @@
+# label_nodes role
+
+This role applies labels defined at inventory level to the OCP cluster nodes.
+
+NOTES:
+ - This role has been tested on on IPI, UPI and SNO installers
+ - Other installers that are require an hosts file should work
+
+## Variables
+
+| Variable               | Default        | Type         | Required    | Description                                                              |
+| ---------------------- | ---------------|------------- | ----------- | -------------------------------------------------------------------------|
+| hosts_list             | []             | List         | No          | A list of host that need its labels applied to OCP nodes                 |
+
+## Role requirements for BGP mode
+  - An Openshift cluster.
+  - Access to the kubeconfig file via the `KUBECONFIG` environment variable
+  - The inventory host should have the labels needed in JSON format
+    ```json
+    [workers:vars]
+    labels={"Location": "Boston", "owner": "OCP operators", "cost_center", "IT"} 
+
+    [ocs_nodes:vars]
+    labels={"cluster.ocs.openshift.io/openshift-storage": ""}    
+    ```
```

Comment:
```suggestion
```toml
[workers:vars]
labels={"Location": "Boston", "owner": "OCP operators", "cost_center", "IT"} 

[ocs_nodes:vars]
labels={"cluster.ocs.openshift.io/openshift-storage": ""}    
```
```